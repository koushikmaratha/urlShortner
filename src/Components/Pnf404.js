import React from 'react';

const Pnf404 = () => {
    return (
        <div className="container d-flex justify-content-center ">
            <h1>404 <i className="fas fa-exclamation"></i> {` No Page Found`}</h1>
        </div>
    );
};

export default Pnf404;