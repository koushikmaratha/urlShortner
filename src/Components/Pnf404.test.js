import React from 'react';
import { shallow } from 'enzyme';
import Pnf404 from './Pnf404';

describe('Testing Pnf404 Component',()=>{
    let wrapper;
    beforeEach(()=>(wrapper=shallow(<Pnf404 />)));

    it('Should check the snapShot',()=>{
        expect(wrapper).toMatchSnapshot();
    })
});