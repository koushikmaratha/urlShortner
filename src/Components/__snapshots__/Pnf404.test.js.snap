// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Pnf404 Component Should check the snapShot 1`] = `ShallowWrapper {}`;
